### PR TITLE
Corrected script for some item

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**:

### CODE REVIEW

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -132061,6 +132164,49 @@ Body:
...
+      bonus bMdef,15;
+      bonus bDelayrate,-10;
+      bonus2 bSkillAtk,"SU_CN_METEOR",10;
+      bonus2 bSkillAtk,"SU_SV_STEMSPEAR",10;
+      bonus bHPGainValue,30*.@r;
+      bonus bSPGainValue,3*.@r;
+      bonus bAspdRate,3*getskilllv("SU_CHATTERING");
+      if (getskilllv("SU_SPIRITOFLAND") == 1) {
+         bonus2 bVariableCastrate,"SU_CN_METEOR",-4*(getskilllv("SU_CN_POWDERING")+getskilllv("SU_CN_METEOR")+getskilllv("SU_NYANGGRASS")+getskilllv("SU_SV_STEMSPEAR")+getskilllv("SU_SV_ROOTTWIST"));
+         bonus2 bVariableCastrate,"SU_SV_STEMSPEAR",-4*(getskilllv("SU_CN_POWDERING")+getskilllv("SU_CN_METEOR")+getskilllv("SU_NYANGGRASS")+getskilllv("SU_SV_STEMSPEAR")+getskilllv("SU_SV_ROOTTWIST"));
```
Comment (Atemo) [inline db/re/item_db_equip.yml:132190]:
```suggestion
         .@val = -4*(getskilllv("SU_CN_POWDERING")+getskilllv("SU_CN_METEOR")+getskilllv("SU_NYANGGRASS")+getskilllv("SU_SV_STEMSPEAR")+getskilllv("SU_SV_ROOTTWIST"));
         bonus2 bVariableCastrate,"SU_CN_METEOR",.@val;
         bonus2 bVariableCastrate,"SU_SV_STEMSPEAR",.@val;
```

File: db/re/item_combo_db.txt [hunk 2]
```diff
@@ -2620,3 +2620,11 @@
...
 480053:4211,{ bonus bMdef,5; bonus bMaxHPrate,10; bonus2 bSubEle,Ele_Neutral,20; }
+20799:18137:1752,{ .@r = getequiprefinerycnt(EQI_GARMENT); bonus2 bSkillAtk,"RA_ARROWSTORM",(5*.@r); bonus2 bSkillAtk,"WM_SEVERE_RAINSTORM",(5*.@r); if(getequiprefinerycnt(EQI_HAND_R)>=10) { bonus bAspd,1; } }
+20799:18138:1754,{ .@r = getequiprefinerycnt(EQI_GARMENT); bonus2 bSkillAtk,"RA_ARROWSTORM",(5*.@r); bonus2 bSkillAtk,"WM_SEVERE_RAINSTORM",(5*.@r); if(getequiprefinerycnt(EQI_HAND_R)>=10) { bonus bAspd,1; } }
+20799:18140:1755,{ .@r = getequiprefinerycnt(EQI_GARMENT); bonus2 bSkillAtk,"RA_ARROWSTORM",(5*.@r); bonus2 bSkillAtk,"WM_SEVERE_RAINSTORM",(5*.@r); if(getequiprefinerycnt(EQI_HAND_R)>=10) { bonus bAspd,1; } }
+20799:18139:1756,{ .@r = getequiprefinerycnt(EQI_GARMENT); bonus2 bSkillAtk,"RA_ARROWSTORM",(5*.@r); bonus2 bSkillAtk,"WM_SEVERE_RAINSTORM",(5*.@r); if(getequiprefinerycnt(EQI_HAND_R)>=10) { bonus bAspd,1; } }
+400048:4804,{ bonus bUseSPrate,-10; bonus bHealPower,10; }
+400048:4803,{ bonus bDelayrate,-10; bonus bHealPower,20; }
+400048:4805,{ bonus bVariableCastrate,-10; bonus bHealPower,5; }
+400048:4850,{ bonus bVariableCastrate,-10; bonus bHealPower,5; }
```
Comment (Atemo) [inline db/re/item_combo_db.txt:2630]: There is a combo Ring of Pazuzu + (another card not implemented), we may add a comment to not forget it eventually
```suggestion
400048:4850,{ bonus bVariableCastrate,-10; bonus bHealPower,5; }
//490098:29371,{ if ((eaclass()&EAJL_THIRD && BaseJob == Job_Wizard) { bonus2 bSkillUseSP,"WL_CRIMSONROCK",50; bonus2 bSkillCooldown,"WL_CRIMSONROCK",-2000; } }
```

File: db/re/item_db_equip.yml (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks

File: db/re/item_combo_db.txt (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_combo_db.txt:2630]: Thanks

**Comment by Atemo** [comment]
Thanks !

**Pull request merged**